frontend/views/containers/proposals/MemberRequest.vue

```diff
@@ -0,0 +1,253 @@
+<template lang='pug'>
+  callout-card(
+    v-if='!hasMemberRequest'
+    :title='L("Member Requests")'
+    :svg='SvgConversation'
+    :isCard='true'
+  )
+
+    i18n(tag='p') In Group Income, you can use an invitation link to add members to the group and invite up to 60 people. Once someone uses that link to join the group, they’ll need to be approved by a member of the group with member approval permissions.
+    i18n.has-text-1(tag='p') There are no open requests right now.
+
+    i18n.c-all-requests.button.is-outlined.is-small(
+      tag='span'
+      @click='toggleHistory'
+    ) See all requests
+
+  // TODO: view without current requests
+  // TODO: button "see all requests"
+  page-section(
+    v-else
+    :title='L("Member Requests")'
+  )
+    .c-all-actions
+      i18n.button.is-outlined.is-small(
+        tag='span'
+        @click='approveAll'
+      ) Approve All
+      i18n.button.is-outlined.is-small(
+        tag='span'
+        @click='rejectAll'
+      ) Reject All
+
+    ul.c-group-list
+      li.c-group-member(
+        v-for='{username, displayName, status, date} in requestsSorted'
+        :data-test='username'
```

Comment:
Can you make this `data-test` a bit more specific to the item? e.g. something like:

```js
:data-test='`request-${username}`'
```